modules/aws/ecs.go

```diff
@@ -0,0 +1,107 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/stretchr/testify/require"
+)
+
+// GetEcsCluster fetches information about specified ECS cluster.
+func GetEcsCluster(t *testing.T, region string, name string) *ecs.Cluster {
+	cluster, err := GetEcsClusterE(t, region, name)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return cluster
+}
+
+// GetEcsClusterE fetches information about specified ECS cluster.
+func GetEcsClusterE(t *testing.T, region string, name string) (*ecs.Cluster, error) {
+	client, err := NewEcsClientE(t, region)
+	if err != nil {
+		return nil, err
+	}
+	input := &ecs.DescribeClustersInput{
+		Clusters: []*string{
+			aws.String(name),
+		},
+	}
+	output, err := client.DescribeClusters(input)
+	if err != nil {
+		return nil, err
+	}
+
+	numClusters := len(output.Clusters)
+	if numClusters != 1 {
+		return nil, fmt.Errorf("Couldn't find ECS cluster %v in region %v", name, region)
```

Comment:
Might be clearer to say: "Expected to find 1 ECS cluster named '%s" in region '%v', but found '%d'

Reply:
Updated, thanks!